src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -161,13 +167,27 @@ def format_historical_data(
     return format_snuba_ts_data(data, query_columns, organization)
 
 
+def get_dataset_from_label(dataset_label: str):
```

Comment:
nice! i really like this method, it's small, focused and throws when stuff is borked. 🎉